html/src/components/terminal/xterm/index.ts

```diff
@@ -146,17 +148,19 @@ export class Xterm {
     @bind
     public open(parent: HTMLElement) {
         this.terminal = new Terminal(this.options.termOptions);
-        const { terminal, fitAddon, overlayAddon } = this;
+        const { terminal, fitAddon, overlayAddon, unicode11Addon } = this;
         window.term = terminal as TtydTerminal;
         window.term.fit = () => {
             this.fitAddon.fit();
         };
 
         terminal.loadAddon(fitAddon);
         terminal.loadAddon(overlayAddon);
+        terminal.loadAddon(unicode11Addon);
```

Comment:
```ts
terminal.loadAddon(new Unicode11Addon());
```
we don't need to keep `unicode11Addon` variable here, since `Unicode11Addon` only implements `ITerminalAddon`, no other APIs are exposed publicly.

Reply:
I did it that way initially and saw other examples of how it being set as a variable. I am not a JS developer so I just tried to copy.

Will make this change for you now.